tests/conftest.py

```diff
@@ -178,11 +178,14 @@ def pytest_configure(config):
         "markers", "system(name): mark test to run with named system"
     )
     config.addinivalue_line(
-        "markers", "long_running(name): mark test that run for a long time (many minutes)"
+        "markers", "long_running: mark test that run for a long time (many minutes)"
     )
     config.addinivalue_line(
         "markers", "credential_file(name): mark tests that require credential file in CREDENTIALS_DIR"
     )
+    config.addinivalue_line(
+        "markers", "master_only: mark tests that works only on Airflow 2.0 / master"
```

Comment:
NIT: maybe we should name it 'airflow_2" or similar rather than master. There will be some period of time that we will still produce backport operators and airflow 2.0 will be already released (and won't be master then)

Reply:
Agree, done!